stablehlo/tests/interpret_negate.mlir

```diff
@@ -0,0 +1,247 @@
+// RUN: stablehlo-interpreter --interpret -split-input-file %s | FileCheck %s
+
+// CHECK-LABEL: Evaluated results of function: negate_op_test_si4
+func.func @negate_op_test_si4() -> tensor<5xi4> {
+  %0 = stablehlo.constant dense<[-8, -1, 0, 1, 7]> : tensor<5xi4>
+  %1 = stablehlo.negate %0 : tensor<5xi4>
+  func.return %1 : tensor<5xi4>
+  // CHECK-NEXT: tensor<5xi4>
+  // CHECK-NEXT: -8 : i4
```

Comment:
Sounds like the spec might need an update? Currently, it says that negation of `-8` should be either `7` or `-9`.

Reply:
acknowledged.